controllers/ibu_controller.go

```diff
@@ -64,7 +64,7 @@ func doNotRequeue() ctrl.Result {
 	return ctrl.Result{}
 }
 
-func requeueWithError(err error) (ctrl.Result, error) {
+func doNotRequeueWithError(err error) (ctrl.Result, error) {
 	// can not be fixed by user during reconcile
 	return ctrl.Result{}, err
 }
```

Comment:
please keep me honest here but i think the name of the func was right e.g 

https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#reconcile-loop
https://cloud.redhat.com/blog/subreconciler-patterns-in-action (original inspiration)


Right now we have 
```
func doNotRequeue() ctrl.Result {
	return ctrl.Result{}
}
``` 
which might not be too clear...basically the only way to doNotRequeue --> `ctrl.Result{}, nil`

Reply:
Thanks for sharing the resources - that is correct. I'll restore to the original version :)